Source/ACE.DatLoader/DatFileType.cs

```diff
@@ -54,7 +54,7 @@ public enum DatFileType : uint
 
         [DatDatabaseType(DatDatabaseType.Cell)]
         [DatFileTypeIdRange(0x01010000, 0x013EFFFF)]
-        EnvCell                    = 3, // DB_TYPE_ENVCELL
+        Cell                    = 3, // DB_TYPE_CELL
```

Comment:
this is going to break third party programs. this shouldn't be changed at this point

technically this range is envcells, so if it's labeled as cell in the pdb, the pdb would be inaccurate there.

Reply:
reverted